simulator-core/src/test/java/com/evbox/everon/ocpp/it/provisioning/GetBaseReportIt.java

```diff
@@ -18,7 +18,7 @@
 
 public class GetBaseReportIt extends StationSimulatorSetUp {
 
-    private static final int NOTIFY_REPORT_VARIABLES_LESS_ONE = 8;
+    private static final int NOTIFY_REPORT_VARIABLES_LESS_ONE = 9;
```

Comment:
what does this magic constant?

Reply:
number of variables in the response, when report is build

Reply:
As far as I understood. But its not my code, maybe something to refactor

Reply:
It's the number of notify report variables (in FullInventory) minus one. Feel free to rename it if the name is confusing, it's only used in 1 or 2 tests in this class.

Reply:
I would like not to do it in this MR. We could create TD for that